README.md

```diff
@@ -1,71 +1,62 @@
-# Noora ⭐️
+# Noora Design System
+
+Noora is Tuist's comprehensive design system that provides consistent UI components and patterns across all platforms and environments.
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-Command Line Interfaces (CLIs), though graphically limited due to terminal capabilities, **can still benefit significantly from well-designed and consistent aesthetics across various commands**. This is a role traditionally filled by design systems in Graphical User Interfaces (GUIs), but it remains largely unexplored in the context of terminals.
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-Tuist's terminal experiences, for a long time, suffered from the lack of a design system, leading to issues like inconsistent spacing, information overload, and difficult-to-parse outputs. This situation motivated us to create SwiftTerminal, **a design system specifically for Swift-powered CLIs.**
+## Domains
 
-Noora offers a set of aesthetically pleasing, customizable, and uniform design system for crafting terminal experiences. It encapsulates our extensive experience in crafting the Tuist CLI. We are delighted to share it with the Swift community, aiming to make building terminal experiences an enjoyable process.
+This monorepo contains multiple packages that implement Noora's design principles:
```

Comment:
Noora is implemented for CLIs and the web to achieve cohesive design across different environments. Noora for CLI is implemented as a Swift package whereas we chose Elixir and Phoenix for the web.